Globally disable Windows ARM32 runs to reduce noise in CoreCLR runs

trylek opened the pull request:
As we still don't have a final resolution w.r.t. a new Windows
queue to run ARM32 tests on, I propose globally disable them before
the issue is resolved. We're tracking the underlying problem in:

https://github.com/dotnet/runtime/issues/1097
https://github.com/dotnet/runtime/issues/1663
https://github.com/dotnet/core-eng/issues/8490

Thanks

Tomas

trylek committed:
Globally disable Windows ARM32 runs to reduce noise in CoreCLR runs

As we still don't have a final resolution w.r.t. a new Windows
queue to run ARM32 tests on, I propose globally disable them before
the issue is resolved. We're tracking the underlying problem in:

https://github.com/dotnet/runtime/issues/1097
https://github.com/dotnet/runtime/issues/1663
https://github.com/dotnet/core-eng/issues/8490

Thanks

Tomas

safern commented on eng/pipelines/common/platform-matrix.yml:
```diff
@@ -223,21 +223,25 @@ jobs:
-        helixQueueGroup: ${{ parameters.helixQueueGroup }}
-        managedTestBuildOsGroup: Windows_NT
-        ${{ insert }}: ${{ parameters.jobParameters }}
+# Currently down as there's no suitable ARM32 pool at the moment, tracked under
```
Is this the best way to disable it? 

We still want to build the installer for `Windows_NT_arm`. What I'm currently doing in my single pipeline PR is to just exclude the test runs but still do builds (which make sense to do).

trylek replied:
Hmm, you're probably right, I'll modify the change as you suggest.

trylek committed:
Address Santi's feedback

Based on Santi's suggestion I have moved the logic for skipping
Windows ARM32 testing to helix_queues_setup - this way we can keep
at least ARM32 builds and just skip the actual testing before
the ARM32 queue is available.

Thanks

Tomas
